cardano/base/templates/base.html

```diff
@@ -63,7 +63,7 @@
       </div>
     </div>
 
-    <script src="{% static 'js/jquery-1.12.0.min.js' %}"></script>
+    <script src="{% static 'js/jquery.min.js' %}"></script>
```

Comment:
Why do you prefer to hide the jquery version? I would prefer to keep it more explicit in order to be able to track the version in case we need to upgrade or generally cross-check the version.